### PR TITLE
update two CVEs

### PR DESCRIPTION
**Pull request by CERT-VDE**

### CODE REVIEW

File: 2021/46xxx/CVE-2021-46380.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
```
Comment (cve-team) [inline 2021/46xxx/CVE-2021-46380.json:1]: {} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'version': {'version_data': [{}]}}]}}]}}

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46380.json:1]: {} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng'}]}]}

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46380.json:1]: {} is not allowed for {'reference_data': [{'refsource': 'CONFIRM'}]}

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/46xxx/CVE-2021-46380.json [hunk 2]
```diff
@@ -1,31 +1,8 @@
...
```
Comment (cve-team) [inline 2021/46xxx/CVE-2021-46380.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46380.json:1]: ('Expecting property name enclosed in double quotes: line 18 column 1 (char 476)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2019/12xxx/CVE-2019-12254.json [hunk 3]
```diff
@@ -1,18 +1,151 @@
 {
```
Comment (cve-team) [inline 2019/12xxx/CVE-2019-12254.json:1]: CNA(s) of user (CERTVDE) do not match owning CNA.

File: 2021/46xxx/CVE-2021-46380.json (hunk 2 above)
Comment (cve-team) [inline 2021/46xxx/CVE-2021-46380.json:1]: CNA(s) of user (CERTVDE) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2019/12xxx/CVE-2019-12254.json (hunk 3 above)
Comment (CERT-VDE) [reply 2019/12xxx/CVE-2019-12254.json:1]: That's a pre-CNA CVE of cert@vde

File: 2021/46xxx/CVE-2021-46380.json (hunk 2 above)
Comment (CERT-VDE) [reply 2021/46xxx/CVE-2021-46380.json:1]: See CVE Request 1242802

**Comment by anthnysingleton** [comment]
Hello @CERT-VDE,

I have gone in and switched these two records over to your Org. Thank you for being patient. 

CVE Content Team

**Comment by CERT-VDE** [comment]
> I have gone in and switched these two records over to your Org. Thank you for being patient.

@anthnysingleton 
Thank you Anthony!

CERT@VDE /jochen

**Pull request merged**